### ISSUE TITLE
[youtube] Need to remove (or replace) `ios_embedded` client

### ISSUE THREAD

**Issue by FlyingDragon0420** [yt-dlp/yt-dlp#10217]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Singapore

### Provide a description that is worded well enough to be understood

I request youtube with extractor_args.
`ydl_opts['extractor_args'] = {
    'youtube': {
        'player_client': ['ios', 'ios_embedded'], 'player_skip': ['android', 'webpage', 'configs', 'js']
    }
}`

When i call extract_info method, 404 error appear in log
ydl.extract_info('https://www.youtube.com/watch?v=C7azxUTSgEM', download=False)

2024-06-20 11:04:50,499 lib.youtube_video_runner:1053 WARNING  [youtube] YouTube said: ERROR - Requested entity was not found.
2024-06-20 11:04:50,499 lib.youtube_video_runner:1053 WARNING  [youtube] HTTP Error 404: Not Found. Retrying (1/3)...
2024-06-20 11:04:50,670 lib.youtube_video_runner:1053 WARNING  [youtube] YouTube said: ERROR - Requested entity was not found.
2024-06-20 11:04:50,670 lib.youtube_video_runner:1053 WARNING  [youtube] HTTP Error 404: Not Found. Retrying (2/3)...
2024-06-20 11:04:50,850 lib.youtube_video_runner:1053 WARNING  [youtube] YouTube said: ERROR - Requested entity was not found.
2024-06-20 11:04:50,851 lib.youtube_video_runner:1053 WARNING  [youtube] HTTP Error 404: Not Found. Retrying (3/3)...
2024-06-20 11:04:51,020 lib.youtube_video_runner:1053 WARNING  [youtube] YouTube said: ERROR - Requested entity was not found.
2024-06-20 11:04:51,020 lib.youtube_video_runner:1053 WARNING  [youtube] Unable to download API page: HTTP Error 404: Not Found (caused by <HTTPError 404: Not Found>)

I thought I couldn't get the result after 404, but I still get the result.
Is there something wrong with it?

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
2024-06-20 11:04:49,531 lib.youtube_video_runner:564 INFO     player_client: ios
2024-06-20 11:04:50,499 lib.youtube_video_runner:1053 WARNING  [youtube] YouTube said: ERROR - Requested entity was not found.
2024-06-20 11:04:50,499 lib.youtube_video_runner:1053 WARNING  [youtube] HTTP Error 404: Not Found. Retrying (1/3)...
2024-06-20 11:04:50,670 lib.youtube_video_runner:1053 WARNING  [youtube] YouTube said: ERROR - Requested entity was not found.
2024-06-20 11:04:50,670 lib.youtube_video_runner:1053 WARNING  [youtube] HTTP Error 404: Not Found. Retrying (2/3)...
2024-06-20 11:04:50,850 lib.youtube_video_runner:1053 WARNING  [youtube] YouTube said: ERROR - Requested entity was not found.
2024-06-20 11:04:50,851 lib.youtube_video_runner:1053 WARNING  [youtube] HTTP Error 404: Not Found. Retrying (3/3)...
2024-06-20 11:04:51,020 lib.youtube_video_runner:1053 WARNING  [youtube] YouTube said: ERROR - Requested entity was not found.
2024-06-20 11:04:51,020 lib.youtube_video_runner:1053 WARNING  [youtube] Unable to download API page: HTTP Error 404: Not Found (caused by <HTTPError 404: Not Found>)
2024-06-20 11:04:52,356 lib.youtube_video_runner:276 INFO     video api info: {"id": "C7azxUTSgEM", "title": "解密时刻：是是非非周恩来(完整版)", "formats": [{"format_id": "sb2", "fo........
```

**Comment by bashonly** [comment]
please show **complete *verbose* output** ***from yt-dlp***

**Comment by coletdjnz** [comment]
The innertube client ios_embedded uses no longer exists so that client won't work

**Comment by FlyingDragon0420** [comment]
> The innertube client ios_embedded uses no longer exists so that client won't work

404 disappear when i remove ios_embedded, where can i get this info?

**Comment by bashonly** [comment]
Fixed in 0e539617a41913c7da1edd74fb6543c10ad727b3
